src/analysis/normed_space/basic.lean

```diff
@@ -333,22 +333,24 @@ begin
   { clear ineq,
 
     have limf': tendsto (λ x, ∥f x - s∥) e (nhds 0) := tendsto_iff_norm_tendsto_zero.1 limf,
-    have limg' : tendsto (λ x, ∥g x∥) e (nhds ∥b∥) := filter.tendsto.comp limg (continuous_iff_tendsto.1 continuous_norm _),
+    have limg' : tendsto (λ x, ∥g x∥) e (nhds ∥b∥) := filter.tendsto.comp limg (continuous_iff_continuous_at.1 continuous_norm _),
+
+    have lim1 := tendsto_mul limf' limg',
+    simp only [zero_mul, sub_eq_add_neg] at lim1,
```

Comment:
This changed from `by simp..` to just `simp`. Either keep the `by` or use `{...}`. Note that `by` guarantess that it solves the goal.

Reply:
I don't think I changed this. A `by` doesn't make sense, and doesn't work. The line doesn't prove the previous goal. Note that the `have` uses `:=`. The author of this proof (you? Patrick?) introduced `lim1` and then simplified it.

Reply:
Indeed I proved this, and `simp` is not proving the `have`, which is proved by the term `tendsto_mul limf' limg'`. This is a typical situation where `tendsto_mul` (or `tendsto_add`) gives a result which need a (non-definitional) simplification before being useful. We really need a tactic computing limits...